pilot/cmd/sidecar-injector/main.go

```diff
@@ -109,26 +111,48 @@ var (
 	}
 )
 
-func patchCert() error {
-	const retryTimes = 6 // Try for one minute.
+// patchCertLoop continually reapplies the caBundle PEM. This is required because it can be overwritten with empty
+// values if the original yaml is reapplied (https://github.com/istio/istio/issues/6069).
+func patchCertLoop() error {
```

Comment:
Can you also reference the tracking issue for watch triggered patch (https://github.com/istio/istio/issues/6451)? e.g.

```golang
// TODO(https://github.com/istio/istio/issues/6451) - only patch when caBundle changes
```

Reply:
Done